### DISCUSSION TITLE
Ranking is not updating the activities points

### DISCUSSION THREAD

**Discussion by LemysLen** [JotaDF/moodle-blockgame#21] [Q&A] [answered]
Olá! Obrigado pelo plugin! Consegui fazer quase tudo dele funcionar, exceto o ranking.

Depois que os estudantes completarma 3 atividades, alguns receberam pontuação no ranking e o resto não. E ficou nisso, como pode ver na figura abaixo.

![image](https://user-images.githubusercontent.com/126628666/229390278-939674e3-439c-4a25-8959-ff36f37efd1d.png)

Outros estudantes possuem notas, mas não apareceu no ranking.

Tentei reavaliar, mudar a pontuação na mão... mas nada atualiza os dados do ranking.

Tem alguma ideia do que pode estar errado?

Obrigado.

**Comment by jotadf** [comment, marked as answer]
I believe I know what's going on! For the student to receive the points he needs to enter the course, by doing this the plugin will recalculate his score! take the test accessing as one of the students who did not receive the score and access the course area!

**Reply by LemysLen** [reply]
That did the trick!

Tried with one student, but all need to sign in to get the scores right? Is there any way to force the update, for the ones that did not enter again?
The first ones to get in the course after the grades were given, wont see the real rank they are in compared to the rest of the class.

**Reply by jotadf** [reply]
The idea is to make the student constantly enter the course! Then his rating will be updated! I haven't made a feature for the teacher or admin to update the scores of all students at once because if the course has too many students it can overload the server!
